modules/react/card/stories/examples/Depth.tsx

```diff
@@ -4,8 +4,9 @@ import {depth} from '@workday/canvas-kit-react/tokens';
 
 export const Depth = () => {
   return (
-    <Card heading="Title" depth={depth.inset}>
-      No Padding
+    <Card depth={depth.inset}>
+      <Card.Header>Title</Card.Header>
+      <Card.Body>No Padding</Card.Body>
```

Comment:
I know this will cause a visual change, but we generally never use inset depth on cards so it's weird to show as an example
```suggestion
    <Card depth={depth[1]}>
      <Card.Header>Title</Card.Header>
      <Card.Body>Less depth</Card.Body>
```